osa/provenance/config/definition.yaml

```diff
@@ -56,21 +56,6 @@ activities:
         description:
             "Create DL1 files for an observation run and subrun"
         parameters:
-            - name: ObservationRun
```

Comment:
Are parameters no longer needed for some of the activities (eg `r0_to_dl1`, `dl1_to_dl2`)?

Reply:
Yes, all parameters given are actually present in the paths of the produced datasets or as session attributes. (i.e. date, run number, software version..)

Reply:
I see, thanks